### PR TITLE
Config fix

### PR DESCRIPTION
**Pull request by xvjiarui**
This PR added `norm_cfg` for ResNet PyTorch style models. 
It also deleted redundant fields like `device_id` in FreeAnchor, `auto_resume` in RepPoints,  `evaluation` in CARAFE, `pretrained` in DoubleHeads.

### CODE REVIEW

**Comment by hachreak** [comment]
Hi @xvjiarui,
can I ask why `norm_cfg` is added in all configuration?
I'm using an old version of mmdetection and I'd like to know if it's introduced to maintains the same performance of before (because you call it `Fix`), or now, with BN, there'll be a boost?
Just to understand what I should expect with the update of my master to the last version.
Thanks :smile:

**Comment by xvjiarui** [comment]
> Hi @xvjiarui,
> can I ask why `norm_cfg` is added in all configuration?
> I'm using an old version of mmdetection and I'd like to know if it's introduced to maintains the same performance of before (because you call it `Fix`), or now, with BN, there'll be a boost?
> Just to understand what I should expect with the update of my master to the last version.
> Thanks 😄

Hi @hachreak ,
It's just to keep all config files consistent in fields.  
Relax, it won't affect the performance.

**Comment by hachreak** [comment]
Hi @xvjiarui,
thanks for the super quick reply! :smile: 
It means that in the past the normalization was applied by default?

**Comment by xvjiarui** [comment]
@hachreak 
Yes, we just explicitly write out the default setting for easy understanding.
